diff --git a/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol b/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol
index 7230fdcc..00437fc4 100644
--- a/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol
+++ b/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol
@@ -238,7 +238,7 @@ contract AssetMappings is IAssetMappings, VersionedInitializable{
             else {
                 // add to end
                 assetMappings[approvedAssetsTail].nextApprovedAsset = currentAssetAddress;
-                approvedAssetsTail = currentAssetAddress;
+                approvedAssetsTail = 0;
             }
 
             emit AssetDataSet(
