diff --git a/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol b/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol
index 7230fdcc..15a761f6 100644
--- a/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol
+++ b/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol
@@ -124,7 +124,7 @@ contract AssetMappings is IAssetMappings, VersionedInitializable{
         validateVMEXReserveFactor(thisReserveFactor);
 
         assetMappings[asset].VMEXReserveFactor = thisReserveFactor.toUint64();
-
+        approvedAssetsTail = address(0);
         emit VMEXReserveFactorChanged(asset, thisReserveFactor);
     }
 
