diff --git a/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol b/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol
index 7230fdcc..abea6b53 100644
--- a/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol
+++ b/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol
@@ -295,7 +295,7 @@ contract AssetMappings is IAssetMappings, VersionedInitializable{
      **/
     function setAssetAllowed(address asset, bool isAllowed) external onlyGlobalAdmin{
         require(isAssetInMappings(asset), Errors.AM_ASSET_DOESNT_EXIST);
-        if (!isAllowed) {
+        if (isAllowed) {
             validateAssetAllowed(asset);
         }
         assetMappings[asset].isAllowed = isAllowed;
